modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm

```diff
@@ -495,6 +495,17 @@
 	item_path = /obj/item/toy/plush/skyrat/chunko/inessa
 	donator_only = TRUE
 
+
+/datum/loadout_item/toys/plushie_chiara
+	name = "Commanding Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_chiara
+	ckeywhitelist = list("srq","superlagg")
+
+/datum/loadout_item/toys/plushie_dan
+	name = "Comfy Fox Plushie"
+	item_path = /obj/item/toy/plush/skyrat/plushie_dan
+	ckeywhitelist = list("srq","superlagg")
+
 //Chunko Fops were donated by SlippyJoe, who requested they are usable by everyone
```

Comment:
this comment should be above the plushies

Reply:
Ah I see what you mean now, moved it up